### DISCUSSION TITLE
modification by

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#17286] [How to] [answered]
Hello,
In my instance Yetiforce CRM 6.4 Service pack V6 I see that most records for example (invoices, payments, accounts, contacts, contracts) in updates and update history, the system shows that my user was the one who modified the registration, I am the administator.

Is this problem?
How is it correct?
I would like this information to really show the user who modifies the registration.

**Comment by rskrzypczak** [comment, marked as answer]
All cron tasks running in the background are performed by the first administrator.
if you are the only administrator, then "by you".
The solution is simple - change the name of the current "administrator" and leave it for administrative work and create a new account for yourself.

**Reply by krseguros** [reply]
thank you
